src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpCodeActions.cs

```diff
@@ -708,7 +708,7 @@ static void Main(string[] args)
             ], cancellationToken: HangMitigatingCancellationToken);
         }
 
-        [IdeFact, Trait(Traits.Feature, Traits.Features.CodeActionsConfiguration)]
+        [IdeFact(Skip = "Temporarily disabled"), Trait(Traits.Feature, Traits.Features.CodeActionsConfiguration)]
```

Comment:
❔ Why is this disabled? Was it failing prior to this change, or because of this change?

Reply:
no.  unintended moving around of commits.  this should work fine.